content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -104,22 +113,15 @@ Pods, Controllers and Services are critical elements to managing Windows workloa
 
 {{< feature-state for_k8s_version="v1.14" state="stable" >}}
 
-Docker EE-basic 18.09+ is the recommended container runtime for Windows Server 2019 / 1809 nodes running Kubernetes. This works with the dockershim code included in the kubelet.
+Docker EE-basic 19.03+ is the recommended container runtime for all Windows Server versions. This works with the dockershim code included in the kubelet.
 
 ##### CRI-ContainerD
 
-{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
-
-ContainerD is an OCI-compliant runtime that works with Kubernetes on Linux. Kubernetes v1.18 adds support for {{< glossary_tooltip term_id="containerd" text="ContainerD" >}} on Windows. Progress for ContainerD on Windows can be tracked at [enhancements#1001](https://github.com/kubernetes/enhancements/issues/1001).
-
-{{< caution >}}
-
-ContainerD on Windows in Kubernetes v1.18 has the following known shortcomings:
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
 
-* ContainerD does not have an official release with support for Windows; all development in Kubernetes has been performed against active ContainerD development branches. Production deployments should always use official releases that have been fully tested and are supported with security fixes.
-* Group-Managed Service Accounts are not implemented when using ContainerD - see [containerd/cri#1276](https://github.com/containerd/cri/issues/1276).
+{{< glossary_tooltip term_id="containerd" text="ContainerD" >}} 1.4.0-beta.1+ can also be used as the container runtime for Windows Kubernetes nodes.
```

Comment:
Perhaps add a hyperlink to https://github.com/containerd/containerd/releases/tag/v1.4.0-beta.1 ?

Reply:
I updated to point to a doc that shows how to configure on a windows node.  does that covers this sufficiently?